language-support/hs/bindings/test/DA/Ledger/Sandbox.hs

```diff
@@ -0,0 +1,123 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Sandbox ( -- Run a sandbox for testing on a dynamically selected port
+    SandboxSpec(..),
+    Sandbox(port),
+    startSandbox,
+    shutdownSandbox,
+    withSandbox
+    ) where
+
+import           Control.Concurrent.Async (race)
```

Comment:
We stopped aligning imports to minimize diffs. But we haven’t gone through the codebase and changed all old imports so you might see some inconsistencies here.

Reply:
In truth we just fixed it in GHC upstream, so are desperately waiting for qualified after the module name! Until then, anything goes.

Reply:
I unaligned my imports. No more `stylish-haskell` for me!